### PR TITLE
Update kdb install instructions

### PR DESCRIPTION
**Pull request by cderb**
update kdb apt install instructions and script

### CODE REVIEW

File: docs/install/install.rst [hunk 1]
```diff
@@ -65,10 +65,9 @@ To install the kernels package for your GPU architecture, use the following comm
 
 .. code:: shell
 
-  apt-get install miopenkernels-<arch>-<num cu>
+  apt-get install miopen-hip-<arch>kdb
```
Comment (amd-jnovotny) [inline docs/install/install.rst:68]: Should we add "sudo" to the command? Would apt be preferrable to apt-get?

**Review by amd-jnovotny — APPROVED** [review]
Docs changes look good.

File: docs/install/install.rst (hunk 1 above)
Comment (cderb) [reply docs/install/install.rst:68]: Currently there is some mixing in the docs for using apt vs sudo apt. I don't really mind either, but some uniformity would be nice. sudo is elevating the user so IMO not the main point we're trying to get across, ex in docker you wouldn't need it. I had just avoided addressing the issue here.

Comment (amd-jnovotny) [reply docs/install/install.rst:68]: Sounds good. We can look at standardization at some later time.

**Review by junliume — APPROVED** [review]

**Pull request merged**